include/jemalloc/internal/lockedint.h

```diff
@@ -80,6 +80,64 @@ locked_dec_u64(tsdn_t *tsdn, malloc_mutex_t *mtx, locked_u64_t *p,
 #endif
 }
 
+static inline bool
+locked_advance_u64(tsdn_t *tsdn, malloc_mutex_t *mtx, locked_u64_t *p,
```

Comment:
If this will live in the `lockedint` code, a more generic (or, I guess, more integer-specific) name might make sense. I.e. `locked_inc_mod_u64` (and probably `interval` -> `modulus`).